client/src/pages/Restaurant/RestaurantView/RestaurantInfo.js

```diff
@@ -0,0 +1,22 @@
+import { formatPhoneNumber } from "../../../utils/formatPhoneNumber";
+
+const classes = {
+  information: {
+    fontSize: "0.9rem",
```

Comment:
Give address text the highlight color (like in restaurant card and restaurant preview)